gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=6.0.0-beta07-SNAPSHOT
```

Comment:
I guess this would still be beta 07 snapshot, right as we did not release the 07 version yet